### PR TITLE
Fix kerberos product tests

### PR DESCRIPTION
**Pull request by arhimondr**

### CODE REVIEW

**Comment by akshatnair** [comment]
LGTM, but check why the build is failing

**Comment by maciejgrzybek** [comment]
lgtm

File: pom.xml [hunk 1]
```diff
@@ -116,6 +116,14 @@
         <module>presto-plugin-toolkit</module>
     </modules>
 
+    <repositories>
```
Comment (ArturGajowy) [inline pom.xml:119]: Why is that needed? Are we planning to merge this upstream, or is it going to be maintained as a differentiator?

**Comment by arhimondr** [comment]
@akshatnair It was some intermittent failure.
